src/passes/filePathMangler.ts

```diff
@@ -1,15 +1,11 @@
-import { ImportDirective, SourceUnit } from 'solc-typed-ast';
```

Comment:
I think we can safely delete this pass now, and move the checkPath to rejectUnsupportedFeatures